src/aat/resources/features/F-029/S-053.td.json

```diff
@@ -0,0 +1,26 @@
+{
+  "title": "must return appropriate negative response when event-id does not exist",
+  "_guid_": "S-053",
+  "_extends_": "F-029_Test_Data_Base",
+
+  "specs": [
+    "an active profile in CCD",
+    "contains a non-existing event-id",
+    "has the 404 return code"
+  ],
+  "request": {
+    "pathVariables": {
+      "etid": "CREATE123"
```

Comment:
maybe you may use a more arbitrary value

Reply:
updated